shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/RewardUpdate.hs

```diff
@@ -86,10 +86,24 @@ import Shelley.Spec.Ledger.TxBody (PoolParams (..))
 -- ===============================================================
 
 -- | The result of reward calculation is a pair of aggregate Maps.
-type RewardAns c =
-  ( Map (Credential 'Staking c) (Set (Reward c)),
-    Map (KeyHash 'StakePool c) Likelihood
-  )
+data RewardAns c
```

Comment:
The change from `(,)` to a `data` with bangs is change that I'm most confident is necessary.